### DISCUSSION TITLE
Remix-Hono-Vite Dockerfile not bundling packages correctly?

### DISCUSSION THREAD

**Discussion by cill-i-am** [rphlmr/remix-galaxy#3] [Q&A] [answered]
Hey, so i'm using remix-hono-vite as a starting point for an app a work and we are trying to dockerise it & deploy it to AWS. But we are coming across an issue where it doesn't seem to be including packages correctly. 

I have copied the DOCKERFILE and replaced the npm commands with pnpm, but aside from that it is the same. Building/running it via pnpm works as expected, but running docker:build exists with an error:
`Error [RollupError]: Could not resolve "../use-latest" from "../../packages/hooks/src/use-resize-observer/index.ts"`. `use-latest` is a hook imported into the `use-resize-observer hook`, which is then imported to the employee package. 

I have a repro available here: https://github.com/technifit/tasker/blob/main/apps/employee/Dockerfile. I'm pretty new to both Vite & docker here, so this could easily be me missing something.

**Comment by rphlmr** [comment]
Hello 👋 
This is the reason why I use npm and not pnpm. I was unable to make it works :/
I will take a look asap (afk) if you are still in trouble.

**Reply by cill-i-am** [reply]
Thanks for the advice! I swapped over to using npm but am still getting the same issue. I forked remix-galaxy and made a test branch to see if i could reproduce it, and it fails consistently with the same error whenever I include the hook: https://github.com/Rocinante89/remix-galaxy/tree/dockerised-package-issue

**Reply by rphlmr** [reply]
Thanks, I will look at this tomorrow or Wednesday and come back to you. Sorry for that, maybe I missed something since the last updates.

**Comment by cill-i-am** [comment]
I’ll keep digging in on my end too. I should hopefully have some time this evening to dig in to it.

And please don’t say sorry, the remix hono playground saved us so much time. Thank you so much for putting it together

**Comment by rphlmr** [comment, marked as answer]
@Rocinante89 I found why after hours of debugging ... 😂

https://github.com/rphlmr/remix-galaxy/pull/5/commits/611c457db0a994de6c270645cc406246ff63dcdd#diff-2f754321d62f08ba8392b9b168b83e24ea2852bb5d815d63e767f6c3d23c6ac5

The regex in the `.dockerignore` (*test*) dropped `use-latest` folder 🤦‍♂️

**Reply by cill-i-am** [reply]
Amazing! The missing ... filter definitely didn't help things either
